src/WebJobs.Extensions.NotificationHub/packages.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.NotificationHubs" version="1.0.3" targetFramework="net45" />
+  <package id="Microsoft.Azure.WebJobs" version="1.1.2-alpha-10267" targetFramework="net45" />
+  <package id="Microsoft.Azure.WebJobs.Core" version="1.1.2-alpha-10267" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="1.8.0.0" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
+  <package id="System.Spatial" version="5.6.2" targetFramework="net45" />
+  <package id="WindowsAzure.ServiceBus" version="2.5.4" targetFramework="net45" />
+  <package id="WindowsAzure.Storage" version="4.3.0" targetFramework="net45" />
+</packages>
```

Comment:
Make sure you add the stylecop package here (the same version that the other projects are using). I saw some parameter spellings that it may complain about. Ping me if you need help figuring out how to suppress some of them.